DockerizeJenkinsfile

```diff
@@ -3,7 +3,7 @@
 pipeline {
     agent {
         docker {
-            image 'repo.ci.build.ge.com:8443/predix-security/uaa-ci-testing-with-docker:latest'
+            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing-with-docker:latest'
```

Comment:
we would need to create a new set of `nodeDetails` in the shared lib just for this jenkinsfile file since this image is different from `uaa-ci-testing`